README.md

```diff
@@ -35,7 +35,7 @@ While Infusion allows for just about any possible extension you can imagine and
 | Setting | Description |
 |---------|-------------|
 | `port`  | _(Optional)_ The HTTP port the server will be hosted on. The default value is 8081. If this is changed, it should also be updated in `Dockerfile`, `docker-compose.dev.yml` and `docker-compose.cloud.yml` |
-| `savingEnabled` | Specifies whether saving stories to the database is allowed. If the value is set to `false` and the matching option in `sjrk.storyTelling.baseTheme.page`, then the site is effectively in "read-only" mode and will behave as a collection of stories rather than an authoring tool. For best results, the values in this config file and the `page` grade should match each other. |
+| `authoringEnabled` | Specifies whether creating and saving stories to the database is allowed. If the value is set to `false` and the matching option in `sjrk.storyTelling.base.page`, then the site is effectively in "read-only" mode and will behave as a collection of stories rather than an authoring tool. For best results, the values all `page` grades should be derived from this config file (this is the case in `storyEdit.html`). |
```

Comment:
I think we've decided it's no longer necessary to define the client side `authoringEnabled` flag in `sjrk.storyTelling.base.page`. The client will now fetch this flag from the server.

Reply:
The value that is ultimately used is the one passed into from the server, but there are still some bits of the templates that are controlled by that flag, as well as at least one IoC reference to it in `storyEdit`, so I've kept it in the `page` grade intentionally. I also intentionally left the reference in the documentation, though it makes sense to remove it from there at this point.

Reply:
Why cannot them use the `authoringEnabled` flag passed into from the server as all page components are instantiated with this server flag passed in in their options?